scripts/check-data-freshness.ts

```diff
@@ -15,6 +15,7 @@ async function checkUrlDataAvailability(
     method: 'get',
     url: link,
     timeout: 10000,
+    validateStatus: () => true,
```

Comment:
What's the purpose of this argument?

Reply:
This allows all non-null, non-2xx statuses to count as being “okay” and therefore have its status returned. What I found was that currently, non-2xx weren’t getting treated as errors with statuses, but returning no status. This allows us to do some handling of returned, non-2xx status codes ourselves.